....E.E....E...E...E..F.E....E...E......F
Time: 0.076
There were 8 errors:
1) test28(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test28(RandoopTest0.java:570)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
2) test27(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test27(RandoopTest0.java:558)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
3) test23(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test23(RandoopTest0.java:502)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
4) test20(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test20(RandoopTest0.java:437)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
5) test17(RandoopTest0)java.lang.NullPointerException
	at RandoopTest0.test17(RandoopTest0.java:304)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
6) test14(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test14(RandoopTest0.java:241)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
7) test10(RandoopTest0)java.io.FileNotFoundException: hi! (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:110)
	at java.io.PrintStream.<init>(PrintStream.java:208)
	at RandoopTest0.test10(RandoopTest0.java:182)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
8) test7(RandoopTest0)java.lang.NullPointerException
	at RandoopTest0.test7(RandoopTest0.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
There were 2 failures:
1) test15(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test15(RandoopTest0.java:279)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
2) test1(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test1(RandoopTest0.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)

FAILURES!!!
Tests run: 31,  Failures: 2,  Errors: 8

Cycle length is4
before
after
Reached this
# of deadlocks detected 0
THE SIZE OF THE TRACE OMEN 40032
Exiting analysis
